CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default Owners
+* @redhat-developer/podman-desktop-reviewers
```

Comment:
it's telling me:
`This CODEOWNERS file contains errors …`

Reply:
Should be good now